### ISSUE TITLE
`put_read_take` test is not being run during tests

### ISSUE THREAD

**Issue by JordanMartinez** [slamdata/purescript-avar#13]
While looking through AVar's test and trying them out myself, I noticed that one of the tests isn't even being run. I'm guessing that is a mistake:
- [previous test](https://github.com/slamdata/purescript-avar/blob/master/test/Main.purs#L58)
- [middle test](https://github.com/slamdata/purescript-avar/blob/master/test/Main.purs#L68)
- [next test](https://github.com/slamdata/purescript-avar/blob/master/test/Main.purs#L80)

In `main`, [the middle one is missing](https://github.com/slamdata/purescript-avar/blob/master/test/Main.purs#L200-L201)

Including that test in main produces a failure

Not sure whether this is related to #9

**Comment by JordanMartinez** [comment]
The tests works if one fixes the string. Rather than `foobazfoobar`, it should be `barfoobazfoo`

**Comment by natefaubion** [comment]
Yeah, I think the string is just wrong. `foobazfoobar` would be the result if the put callback was called after it was taken, which is not correct. I'm not sure why it wasn't included. Maybe because it isn't an interesting test.
